### ISSUE TITLE
Some JLLs don't work on Musl systems - initial-exec TLS resolves to dynamic definition

### ISSUE THREAD

**Issue by jpsamaroo** [JuliaPackaging/Yggdrasil#1418]
On a Linux x86_64 musl system (Alpine Linux), I get the following error when loading `GLFW_jll` (version 3.3.2+0):

```
ERROR: LoadError: LoadError: InitError: could not load library "/home/jpsamaroo/.julia/artifacts/6ffe94a97b8fde49babd75ba7331d5440b3c1a7a/lib/libGLESv1_CM.so"
Error relocating /home/jpsamaroo/.julia/artifacts/6ffe94a97b8fde49babd75ba7331d5440b3c1a7a/lib/libGLESv1_CM.so: _glapi_tls_Current: initial-exec TLS resolves to dynamic definition in /home/jpsamaroo/.julia/artifacts/6ffe94a97b8fde49babd75ba7331d5440b3c1a7a/lib/libGLdispatch.so.0
```

I asked on Slack, and was pointed to [this issue](https://github.com/JuliaLang/julia/issues/36458), but I do not believe this is the same issue. @vtjnash has also indicated that this behavior is not correct, saying on Slack:

> If libGL is explicitly asking the compiler to generate bad code, like libgomp does (instead of letting the compiler decide to emit the code just when valid), yeah, that’d be a bug in the library

It would be great if either the compiler flags could be tuned, or the offending packages patched, to workaround this error.

@giordano @staticfloat

**Comment by giordano** [comment]
@jpsamaroo fixed by #1720?

**Comment by jpsamaroo** [comment]
Yep! I just had to pull down master of a few JLLs and tweak Rmath.jl, and then Distributions.jl loads fine!
